cms/envs/common.py

```diff
@@ -173,6 +173,9 @@
     # How many seconds to show the bumper again, default is 7 days:
     'SHOW_BUMPER_PERIODICITY': 7 * 24 * 3600,
 
+    # Enable Web/HTML Certificates -- be sure to also set in LMS configuration
+    'CERTIFICATES_HTML_VIEW': False,
+
```

Comment:
I removed the pre-existing flag, opting to instead keep the new one because of its proximity to the badges flag.